src/main/java/Lesson6/FilesReader.java

```diff
@@ -0,0 +1,20 @@
+package Lesson6;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class FilesReader {
+    public static void main(String[] args) {
+        try {
+            String pathToFile = "C:\\Users\\Jonatan\\IdeaProjects\\data.txt\\data.txt";
```

Comment:
Użyj ścieżki relatywnej, a nie absolutnej (relative path).
Aktualnie jakbym ja u siebie uruchomił ten program to on nie zadziała, ponieważ mogę nie mieć folderu:

```
C:\\Users\\Jonatan\\IdeaProjects\\
```